File: AdvantageFramework.Core/Database/Entities/VendorRepresentative.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AdvantageFramework.Core.Database.Entities
{
    [Table("VEN_REP")]
    public partial class VendorRepresentative
    {
        [Key]
        [Column("VN_CODE")]
        [StringLength(6)]
        public string VnCode { get; set; }
        [Key]
        [Column("VR_CODE")]
        [StringLength(4)]
        public string VrCode { get; set; }
        [Column("VR_FNAME")]
        [StringLength(30)]
        public string VrFname { get; set; }
        [Column("VR_LNAME")]
        [StringLength(30)]
        public string VrLname { get; set; }
        [Column("VR_MI")]
        [StringLength(1)]
        public string VrMi { get; set; }
        [Column("VR_FIRM_NAME")]
        [StringLength(40)]
        public string VrFirmName { get; set; }
        [Column("VR_ADDRESS1")]
        [StringLength(40)]
        public string VrAddress1 { get; set; }
        [Column("VR_ADDRESS2")]
        [StringLength(40)]
        public string VrAddress2 { get; set; }
        [Column("VR_CITY")]
        [StringLength(20)]
        public string VrCity { get; set; }
        [Column("VR_COUNTY")]
        [StringLength(20)]
        public string VrCounty { get; set; }
        [Column("VR_STATE")]
        [StringLength(10)]
        public string VrState { get; set; }
        [Column("VR_COUNTRY")]
        [StringLength(20)]
        public string VrCountry { get; set; }
        [Column("VR_ZIP")]
        [StringLength(10)]
        public string VrZip { get; set; }
        [Column("VR_TELEPHONE")]
        [StringLength(13)]
        public string VrTelephone { get; set; }
        [Column("VR_EXTENTION")]
        [StringLength(4)]
        public string VrExtention { get; set; }
        [Column("VR_FAX")]
        [StringLength(13)]
        public string VrFax { get; set; }
        [Column("VR_FAX_EXTENTION")]
        [StringLength(4)]
        public string VrFaxExtention { get; set; }
        [Column("EMAIL_ADDRESS")]
        [StringLength(50)]
        public string EmailAddress { get; set; }
        [Column("VR_INACTIVE_FLAG")]
        public short? VrInactiveFlag { get; set; }
        [Column("VR_LABEL")]
        [StringLength(20)]
        public string VrLabel { get; set; }
        [Column("VR_PHONE_CELL")]
        [StringLength(13)]
        public string VrPhoneCell { get; set; }
        [Column("CONTACT_TYPE_ID")]
        public int? ContactTypeId { get; set; }
    }
}

File: AdvantageAPI/APIDbContext/Entities/VendorRepresentative.vb
<Table("VEN_REP")>
Public Class VendorRepresentative

#Region " Constants "



#End Region

#Region " Enum "



#End Region

#Region " Variables "

#End Region

#Region " Properties "

    <Key>
    <Column("VN_CODE", Order:=0, TypeName:="varchar")>
    <MaxLength(6)>
    Public Property VendorCode() As String
    <Key>
    <Column("VR_CODE", Order:=1, TypeName:="varchar")>
    <MaxLength(4)>
    Public Property Code() As String
    <Column("VR_FNAME", TypeName:="varchar")>
    <MaxLength(30)>
    Public Property FirstName() As String
    <Column("VR_LNAME", TypeName:="varchar")>
    <MaxLength(30)>
    Public Property LastName() As String
    <Column("VR_MI", TypeName:="varchar")>
    <MaxLength(1)>
    Public Property MiddleInitial() As String
    <Column("VR_FIRM_NAME", TypeName:="varchar")>
    <MaxLength(40)>
    Public Property FirmName() As String
    <Column("VR_ADDRESS1", TypeName:="varchar")>
    <MaxLength(40)>
    Public Property Address1() As String
    <Column("VR_ADDRESS2", TypeName:="varchar")>
    <MaxLength(40)>
    Public Property Address2() As String
    <Column("VR_CITY", TypeName:="varchar")>
    <MaxLength(20)>
    Public Property City() As String
    <Column("VR_COUNTY", TypeName:="varchar")>
    <MaxLength(20)>
    Public Property County() As String
    <Column("VR_STATE", TypeName:="varchar")>
    <MaxLength(10)>
    Public Property State() As String
    <Column("VR_COUNTRY", TypeName:="varchar")>
    <MaxLength(20)>
    Public Property Country() As String
    <Column("VR_ZIP", TypeName:="varchar")>
    <MaxLength(10)>
    Public Property Zip() As String
    <Column("VR_TELEPHONE", TypeName:="varchar")>
    <MaxLength(13)>
    Public Property Telephone() As String
    <Column("VR_EXTENTION", TypeName:="varchar")>
    <MaxLength(4)>
    Public Property TelephoneExtension() As String
    <Column("VR_FAX", TypeName:="varchar")>
    <MaxLength(13)>
    Public Property Fax() As String
    <Column("VR_FAX_EXTENTION", TypeName:="varchar")>
    <MaxLength(4)>
    Public Property FaxExtension() As String
    <Column("EMAIL_ADDRESS", TypeName:="varchar")>
    <MaxLength(50)>
    Public Property EmailAddress() As String
    <Column("VR_INACTIVE_FLAG")>
    Public Property IsInactive() As System.Nullable(Of Int16)
    <Column("VR_PHONE_CELL", TypeName:="varchar")>
    <MaxLength(13)>
    Public Property CellPhone() As String
    <Column("CONTACT_TYPE_ID")>
    Public Property ContactTypeID() As System.Nullable(Of Integer)
    <NotMapped>
    Public Property VendorName As String
        Get
            VendorName = Vendor.Name
        End Get
        Set(value As String)

        End Set
    End Property

    <ForeignKey("VendorCode")>
    <System.Xml.Serialization.XmlIgnoreAttribute>
    Public Overridable Property Vendor As AdvantageAPI.Vendor

#End Region

#Region " Methods "



#End Region

End Class

